### ISSUE TITLE
Protontricks GUI doesn't work with Proton 5.0-1

### ISSUE THREAD

**Issue by NoXPhasma** [Matoking/protontricks#48]
When I try to open the GUI with protontricks with Proton 5.0-1 it fails with undefined symbol error:

- Distro version: Manjaro Linux
- Proton version: 5.0-1

```
$ protontricks --version
protontricks (1.3.1)

$ winetricks --version
20191224 - sha256sum: afe039a7d72553cb761f0367f9f2085b92af8caf86e025c34bbc1fdd89a1f9ee

$ protontricks 244850 --gui
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
winetricks GUI enabled, using zenity 
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
Using winetricks 20191224 - sha256sum: afe039a7d72553cb761f0367f9f2085b92af8caf86e025c34bbc1fdd89a1f9ee with wine-5.0 and WINEARCH=win64
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
winetricks GUI enabled, using zenity 
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
```

On the official proton repository It has been [suggested](https://github.com/ValveSoftware/Proton/issues/3500#issuecomment-584168295) that protontricks empty LD_LIBRARY_PATH before it calls zenity.

*UPDATE*
I've noticed that my protontricks version was outdated, but even with version 1.4 it's the same:

```
$ pipx upgrade-all 
upgraded package protontricks from 1.3.1 to 1.4 (location: /home/noxp/.local/pipx/venvs/protontricks)

$ protontricks --version   
protontricks (1.4)

$ protontricks 244850 --gui
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
winetricks GUI enabled, using zenity 
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
Using winetricks 20191224 - sha256sum: afe039a7d72553cb761f0367f9f2085b92af8caf86e025c34bbc1fdd89a1f9ee with wine-5.0 and WINEARCH=win64
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
winetricks GUI enabled, using zenity 
zenity: symbol lookup error: /usr/lib/libsoup-2.4.so.1: undefined symbol: g_file_info_get_modification_date_time
```

**Comment by legluondunet** [comment]
Same issue here on Ubuntu 19.10 and Proton 5.0-1

**Comment by Matoking** [comment]
You can use `protontricks --no-runtime <appid>` as a workaround for now. 

---

By default Protontricks uses Steam Runtime and Proton's libraries when running winetricks, and winetricks in turn will use Wine binaries for updating the prefix and zenity for the GUI. This is usually a good thing, since we want the underlying Proton Wine binaries to use Proton's shared libraries -- they were built against those libraries and running them against the system's own libraries may cause problems due to mismatching versions (see issue #3 for earlier discussion).

However, zenity causes a problem here since it's built against the system libraries and not Proton's libraries. Ideally, we would only use Steam Runtime & Proton libraries for binaries inside Proton's `dist/bin` directory and system libraries for everything else, but I'm not sure what is the cleanest way to solve this. Other option is to make the "disable Steam Runtime" option the default, but that's not an ideal solution.

**Comment by Matoking** [comment]
I've added an experimental fix to the `wine_binary_link` branch. You can test it out with pipx:
```
pipx install --force --spec git+https://github.com/Matoking/protontricks.git@wine_binary_link protontricks
```

The fix works by creating a temporary directory with a set of helper scripts named after the Proton's own binaries that force the corresponding binary to be loaded with Steam Runtime in effect. This way, instead of modifying $LD_LIBRARY_PATH for all executables, we prepend $PATH and only target binaries such as `wine` and `wineserver`, which should result in less hassle in overall.

**Comment by NoXPhasma** [comment]
Skipping the runtime works. However, pipx fails to install the `wine_binary_link` branch:

```
pipx install --force --spec git+https://github.com/Matoking/protontricks.git@wine_binary_link protontricks
usage: pipx [-h] [--version]
            {install,inject,upgrade,upgrade-all,uninstall,uninstall-all,reinstall-all,list,run,runpip,ensurepath,completions}
            ...
pipx: error: unrecognized arguments: --spec protontricks
```

**Comment by Matoking** [comment]
It seems newer pipx versions only requires the following command:
```
pipx install --force git+https://github.com/Matoking/protontricks.git@wine_binary_link
```

**Comment by NoXPhasma** [comment]
That command works and also protontricks GUI works with the `wine_binary_link` branch.

**Comment by Matoking** [comment]
I've refined the fix and released version 1.4.1 with the bug fix in place.
